Tests/SwiftBSONTests/ExtendedJSONConversionTests.swift

```diff
@@ -373,6 +373,15 @@ open class ExtendedJSONConversionTestCase: BSONTestCase {
             .to(throwError(errorType: DecodingError.self))
         expect(try BSONTimestamp(fromExtJSON: ["$timestamp": ["t": 1, "i": 2], "extra": "2"], keyPath: []))
             .to(throwError(errorType: DecodingError.self))
+
+        // Comparable testing
```

Comment:
this test file, and all the tests it currently contains, are specifically about converting BSON types to and from extended JSON. I think it would be better to pull this code out into its own test case. maybe in `BSONValueTests.swift`?